Description: Search for tcl.h and switch to tk-dev.
Author: Sergei Golovan <sgolovan@debian.org>
Origin: Debian
Last-Update: 2015-11-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.in
+++ b/configure.in
@@ -141,9 +141,9 @@
 ])
 
 if test "x$with_tcltk" != "xno" ; then
-	AC_SEARCH_LIBS([Tcl_Init], [tcl84 tcl8.4 tcl83 tcl8.3],
-	AC_SEARCH_LIBS([Tk_Init], [tk84 tk8.4 tk83 tk8.3],
-	    [AC_SEARCH_HEADERS(/usr/local/include/tcl.h /usr/local/include/tcl/tcl.h /usr/include/tcl.h /usr/include/tcl8.4/tcl.h /usr/local/include/tcl8.4/tcl.h /usr/include/tcl8.3/tcl.h /usr/local/include/tcl8.3/tcl.h
+	AC_SEARCH_LIBS([Tcl_Init], [tcl tcl84 tcl8.4 tcl83 tcl8.3],
+	AC_SEARCH_LIBS([Tk_Init], [tk tk84 tk8.4 tk83 tk8.3],
+	    [AC_SEARCH_HEADERS(/usr/include/tcl/tcl.h /usr/local/include/tcl.h /usr/local/include/tcl/tcl.h /usr/include/tcl.h /usr/include/tcl8.4/tcl.h /usr/local/include/tcl8.4/tcl.h /usr/include/tcl8.3/tcl.h /usr/local/include/tcl8.3/tcl.h
 		    ,BASE_TCL_DIR)
 		
 		AC_SEARCH_HEADERS("$BASE_TCL_DIR/tk.h" /usr/local/include/tk.h /usr/local/include/tcl/tk.h /usr/include/tk.h /usr/include/tk8.4/tk.h /usr/local/include/tk8.4/tk.h /usr/include/tk8.3/tk.h /usr/local/include/tk8.3/tk.h
